internal/gatewayapi/backendtrafficpolicy.go

```diff
@@ -68,10 +68,38 @@ func (t *Translator) ProcessBackendTrafficPolicies(resources *resource.Resources
 
 	handledPolicies := make(map[types.NamespacedName]*egv1a1.BackendTrafficPolicy)
 
+	gatewayPolicyMap := make(map[types.NamespacedName]*egv1a1.BackendTrafficPolicy)
+
 	// Translate
 	// 1. First translate Policies targeting xRoutes
 	// 2. Finally, the policies targeting Gateways
 
+	// Build gateway policy map
```

Comment:
hey can an extra comment here be adding highlighting the why - to make it easier to access the parent policy targeting the gateway that also directs affects the child resource